app/db/connection_manager.py

```
@@ -0,0 +1,443 @@
+
+This module provides:
+- Unified connection pooling for all database operations
+- Automatic connection retry with exponential backoff
+- Connection health monitoring and auto-recovery
+- Support for both sync and async database operations
+- Environment-based configuration with Supabase optimization
+"""
+
+import os
+import logging
+import asyncio
```

Ruff (0.11.9)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)